pkg/api/testapi/testapi.go

```diff
@@ -257,6 +258,15 @@ func init() {
 			externalTypes:        api.Scheme.KnownTypes(externalGroupVersion),
 		}
 	}
+	if _, ok := Groups[kubeadm.GroupName]; !ok {
```

Comment:
This is ugly. I would prefer this and/or other packages providing APIs to use to do the testing without having to modify this package and register it globally. Way out of scope though.